testing/test_config.py

```diff
@@ -18,3 +20,10 @@ def test_tag_regex(tag, expected_version):
     match = config.tag_regex.match(tag)
     version = match.group("version")
     assert version == expected_version
+
+
+def test_config_from_pyproject(tmpdir):
+    fn = tmpdir / "pyproject.toml"
+    fn.write_text("[tool.setuptools_scm]\nenabled = true\n", encoding="utf-8")
```

Comment:
i would like to suggest write_bytes + a bytes literal at this point

Reply:
In my opinion, unicode literals are more friendly and the intended use here (a6ae830).